plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestSortFields.java

```diff
@@ -0,0 +1,97 @@
+
+        assertParse("comment", sortOrder(builder -> builder.asc("comment")));
+        assertParse("\"comment\"", sortOrder(builder -> builder.asc("comment")));
+        assertParse("\"\"\"another\"\" \"\"quoted\"\" \"\"field\"\"\"", sortOrder(builder -> builder.asc("\"another\" \"quoted\" \"field\"")));
+        assertParse("\"\"\"another\"\" \"\"quoted\"\" \"\"field\"\"\" ASC    NULLS   FIRST  ", sortOrder(builder -> builder.asc("\"another\" \"quoted\" \"field\"")));
+        assertParse("\"\"\"another\"\" \"\"quoted\"\" \"\"field\"\"\" ASC    NULLS   LAST    ", sortOrder(builder -> builder.asc("\"another\" \"quoted\" \"field\"", NullOrder.NULLS_LAST)));
+        assertParse("\"\"\"another\"\" \"\"quoted\"\" \"\"field\"\"\" DESC NULLS FIRST", sortOrder(builder -> builder.desc("\"another\" \"quoted\" \"field\"", NullOrder.NULLS_FIRST)));
+        assertParse(" comment   ", sortOrder(builder -> builder.asc("comment")));
+        assertParse("comment ASC", sortOrder(builder -> builder.asc("comment")));
+        assertParse("  comment    ASC  ", sortOrder(builder -> builder.asc("comment")));
+        assertParse("comment ASC NULLS FIRST", sortOrder(builder -> builder.asc("comment")));
+        assertParse("  comment    ASC     NULLS     FIRST    ", sortOrder(builder -> builder.asc("comment")));
+        assertParse("comment ASC NULLS FIRST", sortOrder(builder -> builder.asc("comment", NullOrder.NULLS_FIRST)));
+        assertParse("     comment   ASC       NULLS       FIRST    ", sortOrder(builder -> builder.asc("comment", NullOrder.NULLS_FIRST)));
+        assertParse("comment ASC NULLS FIRST", sortOrder(builder -> builder.asc("comment", NullOrder.NULLS_FIRST)));
+        assertParse("    comment     ASC    NULLS   FIRST      ", sortOrder(builder -> builder.asc("comment", NullOrder.NULLS_FIRST)));
+        assertParse("comment ASC NULLS LAST", sortOrder(builder -> builder.asc("comment", NullOrder.NULLS_LAST)));
+        assertParse("  comment   ASC    NULLS     LAST    ", sortOrder(builder -> builder.asc("comment", NullOrder.NULLS_LAST)));
+        assertParse("comment DESC", sortOrder(builder -> builder.desc("comment")));
+        assertParse("  comment   DESC  ", sortOrder(builder -> builder.desc("comment")));
+        assertParse("comment DESC NULLS FIRST", sortOrder(builder -> builder.desc("comment", NullOrder.NULLS_FIRST)));
+        assertParse("  comment     DESC  NULLS   FIRST ", sortOrder(builder -> builder.desc("comment", NullOrder.NULLS_FIRST)));
+        assertParse("comment DESC NULLS LAST", sortOrder(builder -> builder.desc("comment", NullOrder.NULLS_LAST)));
+        assertParse("  comment   DESC    NULLS   LAST   ", sortOrder(builder -> builder.desc("comment", NullOrder.NULLS_LAST)));
+        assertParse("comment DESC NULLS LAST", sortOrder(builder -> builder.desc("comment")));
+        assertParse("    comment     DESC   NULLS    LAST   ", sortOrder(builder -> builder.desc("comment")));
+    }
```

Comment:
Can we add also a few negative tests for transforms to showcase that they are currently not supported in Trino?